frappe/public/js/frappe/form/grid.js

```diff
@@ -514,6 +514,7 @@ frappe.ui.form.GridRow = Class.extend({
 		this.wrapper.addClass("grid-row-open");
 		frappe.ui.scroll(this.wrapper, true, 15);
 		me.frm.script_manager.trigger(me.doc.parentfield + "_on_form_rendered");
+		me.frm.cscrpt_manager.trigger('on_form_render', me.doc.doctype, me.doc.name);
```

Comment:
![download 3](https://cloud.githubusercontent.com/assets/1288676/10724457/d941d31c-7bab-11e5-9f80-153cb519cf72.png)

@anandpdoshi this is my use case!

But any developer that need customize a control while the grid form is rendering will pass for the same problem!

A another use case that I can think is, you need render a template in a HTML field, which a information provided by the record, you only will can do it if you create you own patch!

Another detail, yes, it works, but dont take care of pass the reference of children record!